src/models/project.js

```diff
@@ -1,5 +1,29 @@
 const { model, Schema } = require('mongoose');
 
+const Comment = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'users',
+  }, 
+  commentText : String,
```

Comment:
here i think instead of commentText it can be content which is more readable since we will access these like this project.comment.commentText, instead project.comment.content is much much readable

Reply:
Thanks, done